src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,21 @@
+package duke.task;
+
+public class Deadline extends Task {
+    protected String by;
+
```

Comment:
Perhaps consider using variable names that explain the named entity to the reader (value adding) instead or describing the data at a sufficient level of detail